packages/client/src/runtime/core/engines/library/LibraryEngine.ts

```diff
@@ -123,6 +132,26 @@ export class LibraryEngine implements Engine<undefined> {
     }
   }
 
+  async applyPendingMigrations(): Promise<void> {
+    if (TARGET_BUILD_TYPE === 'rn') {
+      await this.start()
+      // @ts-ignore
+      __PrismaProxy.applyPendingMigrations(this.engine, this.datamodel)
+    } else {
+      throw new Error('Cannot call this method from this type of engine instance')
+    }
+  }
+
+  async pushSchema(): Promise<void> {
+    if (TARGET_BUILD_TYPE === 'rn') {
+      await this.start()
+      // @ts-ignore
+      __PrismaProxy.pushSchema(this.engine, this.datamodel)
```

Comment:
Why do we need separate `pushSchema` call? We already passing the `datamodel` on to `__PrismaProxy.create`

Reply:
This is outdated, I removed the push schema call when I removed the dependency on schema-core.

Just as documentation, this was posible because schema-core was included and that used to contain the push function, which would be used for initializing the db. Now the applyMigrations step is always required. For now there are no plans to bring back pushSchema.

I will take care of this.